### ISSUE TITLE
Trichoschisis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7403]
**Preferred term label:**
Trichoschisis

broken or split hairs. Alternating light and dark bands called 'tiger-tail banding', in the hair shaft, found by polarised light microscopy or trichoscopy. 

Is this a synonym for [Tiger tail banding HP:0045055](https://hpo.jax.org/app/browse/term/HP:0045055)

601675  11/11  PMID:9758621

**Comment by pnrobinson** [comment]
Not a synonym
Probably rearrange with 
Abnormal hair shaft morphology

**Comment by pnrobinson** [comment]
Definition

transverse fractures through the hair shafts (trichoschisis). Trichoschisis is characterized by a sharp transverse fracture of the hair shaft.
PMID:23180925

Comment: Hair from individuals with trichothiodistrophy (TTD) exhibits an alternating light and dark (so-called tiger tail) banding pattern with trichoschisis. See figure 2 of PMID:23180925.

**Comment by pnrobinson** [comment]
added
